app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -154,7 +154,9 @@ def scan_job_create(entity)
   end
 
   def raw_scan_job_create(target_class, target_id = nil)
-    target_class, target_id = target_class.class.name, target_class.id if target_class.kind_of?(ContainerImage)
+    if target_class.kind_of?(ContainerImage)
+      target_class, target_id = target_class.class.name, target_class.id
```

Comment:
This was for backward compatibility. It used to be `target` and was replaced by `target_class`, `target_id` to remove queue serialization. This line is for backward compatibility in case we find `target` in queue post upgrade

See discussion https://github.com/ManageIQ/manageiq/pull/13722